A66-otel-stats.md

```diff
@@ -0,0 +1,305 @@
+    Total bytes (compressed but not encrypted) received across all response messages (metadata excluded) per RPC attempt; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.target, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+
+### Client Per-Call Instruments
+
+*   **grpc.client.call.duration** <br>
+    This metric aims to measure the end-to-end time the gRPC library takes to complete an RPC from the application’s perspective. <br>
+    Start timestamp - After the client application starts the RPC. <br>
+    End timestamp - Before the status of the RPC is delivered to the application. <br>
+    If the implementation uses an interceptor then the exact start and end timestamps would depend on the ordering of the interceptors. Non-interceptor implementations should record the timestamps as close as possible to the top of the gRPC stack, i.e., payload serialization should be included in the measurement. <br>
+    *Attributes*: grpc.method, grpc.target, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+### Server Instruments
+
+*   **grpc.server.call.started** <br>
+    The total number of RPCs started, including those that have not completed. <br>
+    *Attributes*: grpc.method, grpc.authority <br>
+    *Type*: counter <br>
+    *Unit*: {call} <br>
+*   **grpc.server.call.sent_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) sent across all response messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.rcvd_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) received across all request messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.duration** <br>
+    This metric aims to measure the end2end time an RPC takes from the server transport’s (HTTP2/ inproc / cronet) perspective. <br>
+    Start timestamp - After the transport knows that it's got a new stream. For HTTP2, this would be after the first header frame for the stream has been received and decoded. Whether the timestamp is recorded before or after HPACK is left to the implementation. <br>
+    End timestamp - Ends at the first point where the transport considers the stream done. For HTTP2, this would be when scheduling a trailing header with END_STREAM to be written, or RST_STREAM, or a connection abort. Note that this wouldn’t necessarily mean that the bytes have also been immediately scheduled to be written by TCP. <br>
+    *Attributes*: grpc.method, grpc.authority, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+## Language Specifics
```

Comment:
I think we need a subsection to describe the overall architecture of the OT plugin in a language-agnostic way, more than just stating the required capabilities.  This needs to describe things like the following:
- The CallTracer approach: An object is created for a call.  The object's methods will be invoked when various events happen over the lifetime of the call.  On the client side, the CallTracer knows about multiple attempts on the same call.  Needs to support more than one CallTracer per call.
- The OT plugin will basically be a way of configuring a CallTracer factory on gRPC clients and servers.  On the client side, the CallTracer will be created by an interceptor; on the server side, it will be a ServerCallTracerFactory.

We do want to describe this generically enough that it applies to all languages (and please feel free to tweak the langauge above as needed to accomplish that).  However, the goal here should be to be specific enough that we can hand this design to a junior engineer to implement and get the right result.